FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.SchemaManagement/Parquet/FhirParquetSchemaValidator.cs

```diff
@@ -0,0 +1,76 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Health.Fhir.Synapse.SchemaManagement.Parquet
+{
+    public class FhirParquetSchemaValidator
+    {
+        public static bool Validate(string schemaKey, FhirParquetSchemaNode rootNode, ref string error)
+        {
+            if (schemaKey == null)
+            {
+                error = "The schema key cannot be null.";
```

Comment:
What about wrapping the error message and bool value into a  ValidationResult { bool Success; string Message }?

Reply:
Sure, updated.